### ISSUE TITLE
ScrollView in Screen with StackPresentation 'modal'

### ISSUE THREAD

**Issue by andredewaard** [software-mansion/react-native-screens#640]
Is it possible to add a ScrollView inside a modal with a refreshControl?

I have this as Native Stack Navigator
```
    <Stack.Navigator screenOptions={{headerShown: false}} mode={'modal'} options={{ animationEnabled: false }}>
      {
        !isAuthenticated ?
          <Stack.Screen name='Authentication' component={AuthRoutes}/> :
          <>
            <Stack.Screen name='Platform' component={PlatformRoutes}/>
            <Stack.Screen name="Profile" component={Profile} options={{ stackPresentation: 'modal', animationEnabled: true }}/>
          </>
      }
    </Stack.Navigator>
```
And inside there is a profile page which also included a list of followers which can be refreshed with the refreshControl, like this:
```
    <ScrollView
      refreshControl={
        <RefreshControl tintColor={themeStyles.color} refreshing={fetchingUsers} onRefresh={onRefresh}/>
      }>
      {
        !initialFetching && !!users && users.length > 0 ?
          users.map((u) => <FollowerFollowingUser toRouteName={'Users'} key={u.id} canUpdate={canUpdateFollowing}
                                                  user={u} styling={themeStyles} {...props}/>) :
          <Text style={{color: themeStyles.secondaryColor, textAlign: 'center', marginTop: 100}}>{emptyStateText}</Text>
      }
    </ScrollView>
```
On pull down to refresh nothing happens to the refreshControl but instead dismisses the modal.

At the app updates in the iOS app store there is a similair situation.

**Comment by WoLewicki** [comment]
Does it work in RN's `Modal` component?

**Comment by WoLewicki** [comment]
I tested it and this commit: https://github.com/facebook/react-native/commit/702ff6d1cdc378815ff1beb563c2c8cc4f30585c should fix the issue. Unfortunately, it is not yet available in RN 0.63. Could you apply the changes made in it and see if the problem disappears for you too? If so, I think we can close it since there is nothing that `RNScreens` can do about it.

**Comment by kacperkapusciak** [comment]
I confirm that the issue is fixed in React Native 0.64.
<details>
<summary>RefreshControl in modal React Native 0.64</summary>


https://user-images.githubusercontent.com/39658211/113726314-6a079300-96f4-11eb-8cd4-07ae9ed248ef.mov



</details>
<details>
<summary>RefreshControl in modal React Native 0.63</summary>

https://user-images.githubusercontent.com/39658211/113726199-51977880-96f4-11eb-94f6-0fc485b736f4.mov

</details>
